Neuron/Sections/Transaction/TransactionConfirmViewController.swift

```diff
@@ -159,7 +159,7 @@ class TransactionConfirmInfoViewController: UIViewController {
             amountLabel.attributedText = attributedText
             fromAddressLabel.text = service.fromAddress
             toAddressLabel.text = service.toAddress
-            gasCostLabel.text = "\(service.gasCost)"
+            gasCostLabel.text = "\(service.gasCost)" + "\(service.token.symbol)"
```

Comment:
Isn't tx fee always in Ether (ETH)? For example if token is 'Abc', tx fee should be something like `0.005ETH`, not `0.005Abc`.